source/configuration_manual/authentication/digest-md5.rst

```diff
@@ -0,0 +1,84 @@
+.. _authentication-digestmd5:
+
+==========
+Digest-MD5
+==========
+
+Digest-MD5 has two things that make it special and which can cause
+problems:
+
+-  Instead of using user@domain usernames, it supports **realms**.
+-  User name and realm are part of the MD5 hash that's used for
+   authentication.
+
+For these and other reasons `Digest-MD5 has been
+obsoleted <https://www.ietf.org/rfc/rfc6331.html>`_ by
+`SCRAM <https://www.ietf.org/rfc/rfc5802.html>`_.
+
+Realms
+^^^^^^
+
+Dovecot v1.0 has problems handling user@domain style usernames with
+Digest-MD5 and with passwords stored in plaintext in the password
+database.
+
+Instead, user@realm is used. Realms are an integral part of Digest-MD5.
+You will need to specify realms you want to advertise to the client in
+the config file:
+
+.. code-block:: none
+
+   auth_realms = example.com another.example.com foo
+
+The realms don't have to be domains. All listed realms are presented to
+the client and it can select to use one of them. Some clients always use
+the first realm. Some clients use your domain name, whenever given more
+than one realm to choose from. Even if this was NOT one of the choices
+you provided (KMail, others?). In both cases the user never sees the
+advertised realms.
+
+You can also set an ``auth_default_realm`` to use when a client gives an
+empty realm. However the client is supposed to use the same realm it
+tells the server for the calculation. So when this option helps, it is
+only for broken clients.
```

Comment:
This is also kind of obsolete text. Also the name of the setting is bad (-> DOP-2834). I think what this text should now say is that attempting to use "auth_default_realm" setting is likely to break DIGEST-MD5 authentication. Also auth_default_realm should be a ref link to the setting.